docker-compose.yml

```
@@ -2,7 +2,7 @@ version: "3"
 services:
   # nosemgrep
   emba:
-    image: embeddedanalyzer/emba:latest
+    image: embeddedanalyzer/emba:pip_env
```

Once the new container is up
We can merge 👍 

```suggestion
    image: embeddedanalyzer/emba:latest
```